### ISSUE TITLE
Get the OMOP Oncology Extension into an official CDM release.

### ISSUE THREAD

**Issue by mgurley** [OHDSI/OncologyWG#118]
Please use the https://github.com/OHDSI/OncologyWG/tree/master/ddl as the source of truth for what should be merged into the CDM.

**Comment by sratwani** [comment]
Per Leadership discussion on 12/17, completing this task will make development and vocabulary tasks much simpler as not having Oncology with the CDM model requires additional code and queries that the larger CDM effort already has mechanism for.

**Comment by mgurley** [comment]
Clair pointed me to the following repository as the place for us to find the format of how we need to document the OMOP Oncology CDM extension structural changes:

**Comment by sratwani** [comment]
@dimshitc Based on @mgurley comments above, can this be resourced and assigned to someone from your side to work on.

**Comment by sratwani** [comment]
@cgreich @rimusia  We were hoping you both could help bring this up with the bigger OMOP modeling group. The recommendation here is to make change to the ‘Drug’ domain definition. The idea behind this is that broad categories like chemotherapy, immunotherapy their concepts should have domain ‘Drug’. We need approval from broader community that the ‘Drug’ domain should have categories as such and not just purely drug related information. @dimshitc could you please confirm that I have stated the problem statement accurately.

**Comment by cgreich** [comment]
Here is the deal with this: The V6 is under stress. We are cleaning it up. Rapid changes are difficult at this point. The idea is that if changes are backwards compatible we would add them, and if they are not (changing something) than not. The Oncology Extension is of the former type (just adding tables and fields). Let me talk to Clair.

**Comment by sratwani** [comment]
The documentation for the Oncology Extension was updated: https://ohdsi.github.io/CommonDataModel/oncology.html

The CDM v6.0 DDL will not include the oncology extension. The full CDM DDL will include the oncology extension once v6.1 is released. It cannot be added to v6.0 because v6.0 was already released. The documentation has been added to the CDM website (https://ohdsi.github.io/CommonDataModel/) under proposals -> accepted -> Oncology CDM. The link will include documentation relating to the tables, an explanation that it will be fully added to v6.1, a link to the github branch with the DDL, and a link to the wiki with the instructions

**Comment by sratwani** [comment]
The documentation for the Oncology Extension was updated: https://ohdsi.github.io/CommonDataModel/oncology.html

**Comment by dimshitc** [comment]
But what about this page
https://github.com/OHDSI/CommonDataModel/wiki
?
I think this issue should be closed only after we see it there
